### DISCUSSION TITLE
Could you please assist me with the problem I'm encountering when testing with Jest using 'nativewind' version '3.0.0-next.34'?

### DISCUSSION THREAD

**Discussion by viabells-winko** [nativewind/nativewind#562] [Q&A]
![image](https://github.com/marklawlor/nativewind/assets/140389613/cf861acb-7747-4a06-821a-4c7e6ce27b53)

**Comment by patlux** [comment]
For me it was the missing mock for react-native-svg-transformer: https://github.com/kristerkari/react-native-svg-transformer?tab=readme-ov-file#usage-with-jest

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
